UI/ObjectListWnd.cpp

```diff
@@ -2567,8 +2567,8 @@ void ObjectListWnd::ObjectRightClicked(GG::ListBox::iterator it, const GG::Pt& p
             auto produce_building_action = [this, entry, app, cur_empire, &focus_ship_building_common_action]() {
                 std::string bld = entry.first;
                 bool needs_queue_update(false);
-                for (const auto& entry : m_list_box->Selections()) {
-                    ObjectRow *row = dynamic_cast<ObjectRow *>(entry->get());
+                for (const auto& selection_it : m_list_box->Selections()) {
```

Comment:
The type of `Selections` is a container of `iterator`, but in this case the variable `entry` -> `selection_it` isn't an iterator over the container `Selections`, so I wouldn't name it with `_it` in the name.

Reply:
Changed to `selection` (consistent with previous for loop)